app/src/main/java/com/locoquest/app/dto/Benchmark.kt

```diff
@@ -8,11 +8,11 @@ data class Benchmark(
     val name: String,
     val lat: String,
     val lon: String,
-    val ellipHeight: String,
+    val ellipseHeight: String,
     val posDatum: String,
     val posSource: String,
     val posOrder: String,
-    val orthoHt: String,
+    val orthoHeight: String,
```

Comment:
Renaming variables to be a bit more clear

https://kotlinlang.org/docs/coding-conventions.html#property-names

Reply:
Good.